### DISCUSSION TITLE
Command not found - nofity

### DISCUSSION THREAD

**Discussion by FrancescoSaverioZuppichini** [projectdiscovery/notify#218] [Q&A] [answered]
Hello there,

I have to go installed on my system, I've installed `notify` with `go install -v github.com/projectdiscovery/notify/cmd/notify@latest` but as soon as I try to type `notify` I got `Command 'notify' not found, but can be installed with:` both with `bash` and `zsh`. I am not a go developer but I don't understand what I did wrong

Thanks.

Cheers,

Fra

**Comment by robdew** [comment, marked as answer]
add `~/go/bin` to your $PATH

**Reply by jarek-bir** [reply]
Ad go environment to your  path .

export PATH=$PATH:/usr/local/go/bin
